src/recoilState/index.ts

```diff
@@ -1,6 +1,5 @@
-export * from './appState';
+export * from './baseState';
```

Comment:
fwiw, i would find the code easier to follow if you didn't have this extra level of indirection, and just imported directly from the original files.

Reply:
hm... I may change this - I see what you mean

Reply:
Done